root/usr/sbin/hotsync

```diff
@@ -122,6 +122,14 @@ cat >> ${INCLUDE_FILE} << EOF
 /etc/yum.conf
 EOF
 
+#copy nsdc machine
+if rpm -q --quiet nethserver-dc; then
```

Comment:
Can I implement an equivalent callback script in nethserver-dc RPM? Can I put a (say) `/etc/hotsync.d/nethserver-dc` script and append the `/var/lib/machines/nsdc` line to `${INCLUDE_FILE}` from nethserver-dc?

Reply:
Yes, it can be here or in a script in /etc/hotsync.d/. But all other plugin scripts are in nethserver-hotsync package. I wrote this in main file because dc is like a "core" part, but this choice is highly personal and I'm not really sure about that

Reply:
When hotsync API will be stable, we can move each plugin to its respective package. In the meanwhile, let's move the code above to a dedicated nsdc "plugin" script (no "core" approach)